stpmex/exc.py

```
@@ -86,3 +86,7 @@ class AccountDoesNotExist(StpmexException):
 
 class InvalidAmount(StpmexException):
     """El monto es inválido para una de las instituciones"""
+
+
+class BlockedInstitution(StpmexException):
```

When `BlockedInstitution` inherit from `pydantic.errors.PydanticValueError` pydantic wraps it with `pydantic.error_wrappers.ValidationError` so original exception is lost. And the expected exception would be `ValidationError` instead of `BlockedInstitution`.

---

That's the intended effect. That way, all validation errors are raised at the same time